docs/content/Configuration/VPC/Connecting-with-a-VPC-AWS.mdx

```diff
@@ -5,8 +5,8 @@ permalink: /cloud/configuration/connecting-with-a-vpc/aws
 
 ## Prerequisites
 
-To allow Cube Cloud to connect to a VPC on AWS, the following information is
-required:
+To allow Cube Cloud to connect to a [VPC on AWS][aws-docs-vpc], the following
+information is required:
```

Comment:
@hassankhan I _love_ that you added a link to AWS docs that provides the necessary context!